crates/gossip/src/respond/accept.rs

```diff
@@ -130,6 +130,8 @@ impl K2Gossip {
                     DhtSnapshotNextAction::NewSnapshot(snapshot) => {
                         match snapshot {
                             DhtSnapshot::DiscSectors { .. } => {
+                                tracing::info!(?accept.session_id, "Found a disc mismatch, starting to compare sectors");
```

Comment:
You can normally only see what route gossip sync takes at debug. I'd like to know with less output.